ui/shared/utils/constants.js

```diff
@@ -1041,6 +1041,29 @@ export const REVIEW_TAG_NAME = 'Review'
 export const KNOWN_GENE_FOR_PHENOTYPE_TAG_NAME = 'Known gene for phenotype'
 export const DISCOVERY_CATEGORY_NAME = 'CMG Discovery Tags'
 export const MME_TAG_NAME = 'MME Submission'
+export const SUMMARY_PAGE_SAVED_VARIANT_TAGS = [
```

Comment:
it is confusing to have a page-specific constant in the global constants list, exspecially sicnce its used in other forms elsewhere. It is better to have a single `VARIANT_TAGS` constant and then if there need to be different tag options on different pages you can filter out the ones you don;t want for that specific page

Reply:
The dropdown tag options for the summary page and project page are different.
On the summary page, they are as the following figure:
<img src=https://github.com/broadinstitute/seqr/assets/33550059/b65a1628-a82c-4522-b04a-efc3b189558a width=200 height=400 />
On the project pages, they are:
<img src=https://github.com/broadinstitute/seqr/assets/33550059/50e87fec-d696-4053-9e3b-c890c98223f4 width=200 height=500 />

Reply:
there should not be a page specific constant in the shared constants file. If things are different on different pages, there should be page specific constants in the page specific constants files. If this constant is used in multiple places, then it should not be named `SUMMARY_PAGE_SAVED_VARIANT_TAGS` it should just be `SAVED_VARIANT_TAGS`. If it is just used in one place for one page, then it should be in a different file

Reply:
Okay.

Reply:
Move the constant into the SummaryData folder.